pyroomacoustics/tests/test_moving_sound.py

```diff
@@ -0,0 +1,42 @@
+import numpy as np
+
+import pyroomacoustics as pra
+
+
+def test_simulation_output_shape():
```

Comment:
This is good as a basic test, but it would be nice to have some tests for the correctness of the time-varying convolution too.
One test for that is that the result of a time-varying convolution where all the filters are the same should be the same as a single long convolution.